projects/centre_console/src/power_fsm.c

```diff
@@ -64,6 +65,8 @@ static void prv_state_fault_output(Fsm *fsm, const Event *e, void *context) {
   } else {
     power_fsm->previous_state = POWER_STATE_OFF;
     event_raise(CENTRE_CONSOLE_POWER_EVENT_OFF, 0);
+    CAN_TRANSMIT_DISCHARGE_PRECHARGE();
+    critical_section_start();
```

Comment:
You also need an infinite loop (like a literal `while (1) {}`). Also, this is such a bonkers, out-of-place move that I'd give the BPS fault handling its own function, and it should be heavily commented - say why we're discharging precharge and why we're locking up centre console.

Reply:
Oh if I do that, a test will never complete. Should I just rid of that part then?

Reply:
Yeah don't try testing it, it'll be more of a pain than it's worth